hack/dev-push-hyperkube.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+# Copyright 2014 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script will build and push the hyperkube image.
+# This is useful for rapidly testing changes in a cluster that is
+# powered by hyperkube. You can configure the cluster to use a
```

Comment:
Can you add more details to the comment starting at "You can configure"? For example
- how do you configure the cluster to use a hyperkube tag?
- what does "pulling the tag on the nodes" mean? was this an abbreviation for "using 'docker pull' to pull the tagged hyperkube image you built onto each node" ? If so, please write out the more complete wording.